__tests__/test.js

```diff
@@ -92,6 +93,40 @@ describe("GET /api/articles ", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments ", () => {
+  test(".../:article_id/comments returns all comments for given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then((res) => {
+        expect(res.status).toBe(200);
+        expect(res.body.length).toBeGreaterThan(0);
```

Comment:
Can we be more specific? We know what the data looks like 👍

Reply:
Again, can you assert that the object that comes back is a article that has had it's votes changed?